examples/css/style.css

```diff
@@ -57,26 +60,28 @@ body {
 .scatterplot .erase {
   justify-content: center;
   align-items: center;
+  height: 30px;
+}
+.scatterplot .erase:focus a {
+  outline: 0;
+}
+.scatterplot .erase:hover .erase-fill-element,
+.scatterplot .erase:focus .erase-fill-element {
+  fill: #5a3ec8;
 }
 
 .scatterplot .erase svg {
   cursor: pointer;
 }
 
 .scatterplot .menuEl, .scatterplot .menuEl2 {
```

Comment:
Is there a convention for things that are `kebab-cased` and `camelCased`? noticed some of the selectors follow the former, some the latter.